libs/sqlite/std/hl/_std/sys/db/Sqlite.hx

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (C)2005-2016 Haxe Foundation
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+package sys.db;
+import haxe.crypto.BaseCode;
+
+private typedef SqliteConnectionHandle = hl.types.NativeAbstract<"sqlite_database">;
+private typedef SqliteResultHandle = hl.types.NativeAbstract<"sqlite_result">;
+
+@:hlNative("sqlite")
+private class SqliteLib
+{
+	public static function connect( path : hl.types.Bytes ) : SqliteConnectionHandle { return null; }
+	public static function close( c : SqliteConnectionHandle ) : Void { }
+	public static function request( c : SqliteConnectionHandle, sql : hl.types.Bytes ) : SqliteResultHandle { return null; }
+	public static function last_id( c : SqliteConnectionHandle ) : Int { return 0; }
+	public static function result_next( c : SqliteResultHandle ) : hl.types.NativeArray<Dynamic> { return null; }
+	public static function result_get( c : SqliteResultHandle, n : Int ) : hl.types.Bytes { return null; }
+	public static function result_get_int( c : SqliteResultHandle, n : Int ) : Int { return 0; }
+	public static function result_get_float( c : SqliteResultHandle, n : Int ) : Float { return .0; }
+	public static function result_get_length( c : SqliteResultHandle ) : Int { return 0; }
```

Comment:
result_get_int/float does not seem to be correctly typed, they should be Null no ? (or change the bindings)
result_get should return Dynamic too